Bottom: 8f7410277296302eacbcc318872851931ebc4b55
Top:    15a359a059955aaf0d2b867d30be565ac7af5b74
Author: Jon Smirl <jonsmirl@terra>
Date:   2012-03-19 23:01:41 -0400

Make sure mmc stop command is fully initialized before issuing the parent command


---

diff --git a/drivers/mmc/host/lpc31xx_mmc.c b/drivers/mmc/host/lpc31xx_mmc.c
index f04d4c7..31b8f78 100644
--- a/drivers/mmc/host/lpc31xx_mmc.c
+++ b/drivers/mmc/host/lpc31xx_mmc.c
@@ -611,11 +611,10 @@ static void lpc313x_mci_start_request(struct lpc313x_mci *host,
 	if (data) //we may need to move this code to mci_start_command
 		lpc313x_mci_submit_data(host, data);
 
-	lpc313x_mci_start_command(host, cmd, cmdflags);
-
 	if (mrq->stop) 
 		host->stop_cmdr = lpc313x_mci_prepare_command(slot->mmc, mrq->stop);
-	
+
+	lpc313x_mci_start_command(host, cmd, cmdflags);
 }
